collector/container.go

```diff
@@ -184,8 +184,7 @@ func (c *ContainerMetricsCollector) collect(ch chan<- prometheus.Metric) (*prome
 			log.Error("err in fetching container Statistics: ", containerId, err)
 			continue
 		}
-		// HCS V1 is for docker runtime. Add the docker:// prefix on container_id
-		containerId = "docker://" + containerId
+		containerId = getContainerIdWithPrefix(containerDetails)
```

Comment:
This wasn't introduced by your change, but I think we should avoid mutating the state of the `containerId` variable.
Perhaps a `containerPrefix := getContainerIdWithPrefix(containerDetails)` could be used instead?

Reply:
I had the same thought :smile:   Will make this update.

Reply:
Updated